lib/r_tree.dart

```diff
@@ -26,12 +26,6 @@
 /// rectangles or polygons." - http://en.wikipedia.org/wiki/R-tree
 library r_tree;
 
-import 'dart:math';
-
-part 'src/r_tree/leaf_node.dart';
-part 'src/r_tree/non_leaf_node.dart';
-part 'src/r_tree/node.dart';
-part 'src/r_tree/quickselect.dart';
-part 'src/r_tree/r_tree.dart';
-part 'src/r_tree/r_tree_datum.dart';
-part 'src/r_tree/r_tree_contributor.dart';
+export 'src/r_tree/r_tree.dart';
+export 'src/r_tree/r_tree_datum.dart';
+export 'src/r_tree/r_tree_contributor.dart';
```

Comment:
Dart recommends using `show` so it's clear what symbols are exported

https://dart.dev/guides/libraries/create-packages

> Directly under lib, the main library file, shelf.dart, exports API from several files in lib/src. To avoid exposing more API than intended—and to give developers an overview of the entire public API of the package—shelf.dart uses show to specify exactly which symbols to export:

Reply:
yeah I will definitely be using that

Reply:
already had it just forgot to push the commit :D